From 26bf80befbae90d20be23c2f4200486fdefd4a0c Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Tue, 30 May 2017 20:27:44 +0200
Subject: [PATCH 2/2] Fix build with Qt 5.5

These are obvious typing mistakes that lead to compiler errors when
building with Qt 5.5.1 or earlier.

Differential Revision: https://phabricator.kde.org/D5986
---
 src/viewarea.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/viewarea.cpp b/src/viewarea.cpp
index 8623b8b..d6723cb 100644
--- a/src/viewarea.cpp
+++ b/src/viewarea.cpp
@@ -2032,7 +2032,7 @@ KDE_NO_EXPORT void ViewArea::syncVisual () {
     int h = (int)(height() * devicePixelRatioF());
 #else
     int w = width();
-    int h = heigth();
+    int h = height();
 #endif
     IRect rect = m_repaint_rect.intersect (IRect (0, 0, w, h));
 #ifdef KMPLAYER_WITH_CAIRO
@@ -2156,7 +2156,7 @@ KDE_NO_EXPORT void ViewArea::updateSurfaceBounds () {
     int devicew = (int)(width() * devicePixelRatioF());
     int deviceh = (int)(height() * devicePixelRatioF());
 #else
-    int devicew = width, deviceh = height()
+    int devicew = width(), deviceh = height();
 #endif
     Single x, y, w = devicew, h = deviceh;
     h -= m_view->statusBarHeight ();
-- 
2.12.3

